### PR TITLE
Bugfix: Support Python 3.4, decode from base64 when converting messag…

### PR DESCRIPTION
**Pull request by alukach**
Bugfix: Support Python 3.4, decode from base64 when converting message to Python.

This is cleanup from issues I found with #693.

### CODE REVIEW

**Comment by georgepsarakis** [comment]
Thanks for opening this PR, I will try to look at it ASAP.

File: kombu/async/aws/sqs/message.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ class AsyncMessage(BaseAsyncMessage):
 
     def encode(self, value):
         """Encode/decode the value using Base64 encoding."""
-        return base64.b64encode(value).decode('utf-8')
+        return base64.b64encode(value.encode()).decode()
```
Comment (georgepsarakis) [inline kombu/async/aws/sqs/message.py]: A suggestion here:

```
# Python 2/3 compatibility
from kombu.five import bytes_if_py2
# base64 results to always ASCII character output
return base64.b64encode(bytes_if_py2(value)).decode('ascii')
```

Comment (alukach) [inline kombu/async/aws/sqs/message.py]: Reading the code, I think that would actually be the opposite of what we want:

```
    def bytes_if_py2(s):
        """Convert str to bytes if running under Python 2."""
        return s
```

We want to input [`string` data if using Python 2](https://docs.python.org/2/library/base64.html#base64.b64encode) and to input [`bytes` data if using Python 3](https://docs.python.org/3.4/library/base64.html#base64.b64encode).

`.encode()` returns `string` in Python 2 and `bytes` in Python 3.  I'm pretty new to Python 3, so I may be missing something.  Maybe there's a better way to do this?

Comment (alukach) [inline kombu/async/aws/sqs/message.py]: @georgepsarakis will merge when I have your OK

Comment (georgepsarakis) [inline kombu/async/aws/sqs/message.py]: You are right. I think there is a suitable function that also handles Python 2/3 compatibility:

```
from kombu.utils.encoding import str_to_bytes
```

Comment (alukach) [inline kombu/async/aws/sqs/message.py]: @georgepsarakis Good catch, fixed.

**Pull request merged**